src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -257,25 +234,6 @@ private void setupListeners() {
       updateAttacker(null);
       setWidgetActivation();
     });
-    calculateButton.addMouseMotionListener(new MouseMotionListener() {
-      @Override
-      public void mouseDragged(final MouseEvent e) {}
-
-      @Override
-      public void mouseMoved(final MouseEvent e) {
-        final String memoryAvailable = "<br/>Percentage of memory available: "
-            + String.format("%.2f", (percentageOfFreeMemoryAvailable() * 100)) + "% <br/>Free memory available: "
-            + (freeMemoryAvailable() / (1024 * 1024)) + "MB <br/>Maximum allowed memory: "
-            + (Runtime.getRuntime().maxMemory() / (1024 * 1024)) + "MB </html>";
-        if (calculateButton.isEnabled()) {
-          calculateButton.setToolTipText("<html>Data copying finished. " + memoryAvailable);
-        } else {
-          calculateButton.setToolTipText("<html>If this is taking forever to enable, it means "
-              + "<br/>you do not have enough memory to copy the data quickly! "
-              + "<br/>Consider increasing the max memory for TripleA. " + memoryAvailable);
-        }
-      }
-    });
```

Comment:
I don't think we should try explaining to users what RAM is in this context.
If users run out of memory during this, we might want to consider removing the `-Xmx1G` flag in the vmoptions file, and just leave the minimum option there.

Reply:
I support the removal.

Seems unlikely to run out of memory unless concurrency is set really high... Even then, it's hard to see that happening. If memory is not being reclaimed between iterations then we simply have a memory leak.

Reply:
I think there were two cases for this feature with hovertext:
1. Developer hack to show memory between runs
2. Slow game data loads, user may be hovering over a disabled button wondering why it is not yet enabled.